docs/fiddles/windows/manage-windows/New Window Fiddle/package.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I think as mentioned [here ](https://github.com/electron/electron/issues/20442) the package.json wasn't supposed to be committed.

> Add the contents of your Fiddle (main.js, renderer.js , and index.html but not package.json)

Reply:
Ack, thanks for the suggestion